### PR TITLE
New Validated Patterns tiers 

### PR DESCRIPTION
**Pull request by abhatt-rh**
Based on #319 

Issue: https://issues.redhat.com/browse/TELCODOCS-1521

Preview: http://352.docs-pr.validatedpatterns.io/learn/workflow/

### CODE REVIEW

**Comment by mbaldessari** [comment]
You can preview this docs PR at http://352.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

File: content/learn/tested.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+[id="must-tested-tier"]
+=== Must
+
+A {tested} pattern must continue to meet the following criteria to remain in the {tested} tier:
+
+* A {tested} pattern must conform to the common technical link:/requirements/implementation/[implementation requirements]
+* A {tested} pattern must be meaningful without specialized hardware, including flavors of architectures not explicitly supported.
++
+Qualification is a {solution-name-upstream} TOC decision with input from the pattern owner.
+//AI: What's TOC?
```
Comment (beekhof) [inline content/learn/tested.adoc:49]: Technical Oversight Committee

File: content/learn/tested.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* A {tested} pattern must have their implementation reviewed by the patterns team to ensure that it is sufficiently flexible to function across a variety of platforms, customer environments, and any relevant verticals.
+* A {tested} pattern must include a standardized architecture drawing, created with (or at least conforming to) the PAC tooling
+//AI: What's PAC
```
Comment (beekhof) [inline content/learn/tested.adoc:52]: Portfolio Architecture Council

File: content/learn/tested.adoc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+A {tested} pattern must continue to meet the following criteria to remain in the {tested} tier:
+
+* A {tested} pattern must conform to the common technical link:/requirements/implementation/[implementation requirements]
+* A {tested} pattern must be meaningful without specialized hardware, including flavors of architectures not explicitly supported.
++
+Qualification is a {solution-name-upstream} TOC decision with input from the pattern owner.
+//AI: What's TOC?
+* A {tested} pattern must have their implementation reviewed by the patterns team to ensure that it is sufficiently flexible to function across a variety of platforms, customer environments, and any relevant verticals.
+* A {tested} pattern must include a standardized architecture drawing, created with (or at least conforming to) the PAC tooling
```
Comment (beekhof) [inline content/learn/tested.adoc:51]:
```suggestion
* A {tested} pattern must include a standardized architecture drawing, created with (or at least conforming to) the standard Patterns tooling
```

File: content/learn/implementation.adoc [hunk 4]
```diff
@@ -2,73 +2,82 @@
...
+=== Must
 
-. Patterns *MUST* use a standardized https://github.com/validatedpatterns/common/tree/main/clustergroup[clustergroup] Helm chart, as the initial OpenShift GitOps application that describes all namespaces, subscriptions, and any other GitOps applications which contain the configuration elements that make up the solution.
-. Managed clusters *MUST* operate on the premise of "`eventual consistency`" (automatic retries, and an expectation of idempotence), which is one of the essential benefits of the GitOps model.
-. Imperative elements *MUST* be implemented as idempotent code stored in Git
+. Patterns must include one or more Git repositories in a publicly accessible location, containing configuration elements that can be consumed by the {rh-gitops} Operator without supplying custom Argo CD images.
+. Patterns must be useful without all content stored in private Git repositories.
+. Patterns must include a list of names and versions of all the products and projects that the pattern consumes.
+. Patterns must be useful without any sample applications that are private or that lack public sources.
+//AI:why application was styled that way
```
Comment (beekhof) [inline content/learn/implementation.adoc:36]: I was probably using it as a Noun elsewhere in the list

File: content/learn/implementation.adoc [hunk 5]
```diff
@@ -2,73 +2,82 @@
...
-Additional requirements specific to the implementation for all Community, and Validated patterns
+The requirements are categorized as follows:
 
-[id="must"]
+Must::
+These are non-negotiable, core requirements that must be implemented.
+Should::
+These are important but not critical; their implementation enhances the pattern.
+Can::
+These are optional or desirable features, but their absence does not hinder the implementation of a  pattern.
```
Comment (danmacpherson) [inline content/learn/implementation.adoc:27]: Additional space.

```suggestion
These are optional or desirable features, but their absence does not hinder the implementation of a pattern.
```

Also this list is a good definition of the requirement categories. Should we include these at the beginning of the pages for the other tiers as well?

File: content/learn/maintained.adoc [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+. A {maintained} pattern must only make use of components that are either supported, or easily substituted for supportable equivalents, for example, HashiCorp vault which has community and enterprise variants.
+* A {maintained} pattern must *not* rely on functionality in tech-preview, or hidden behind feature gates.
+* A {maintained} pattern must have their architectures reviewed by the Product Manager (PM), Technical Product Manager (TPM), or Technical Marketing Manager (TMM) of each {redhat} product they consume to ensure consistency with the product teams` intentions and roadmaps.
+* A {maintained} pattern must include a presentation slides oriented around the business problem being solved and intended for use by the field to sell and promote the solution.
+* A {maintained} pattern must include test plan automation that runs on every change to the pattern, or a schedule no less frequently than once per week.
+* A {maintained} pattern must be tested on all currently supported {rh-ocp} extended update support (EUS) releases.
+* A {maintained} pattern must fix breakage in timely manner.
+* A {maintained} pattern must document their support policy.
++
```
Comment (danmacpherson) [inline content/learn/maintained.adoc:56]: Is this + necessary?

File: content/learn/maintained.adoc [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+The individual products used in a {solution-name-upstream} are backed by the full {redhat} support experience conditional on the customer's subscription to those products, and the individual products`s support policy.
++
+Additional components in a {solution-name-upstream} that are not supported by {redhat}; for example, Hashicorp Vault, and Seldon Core, require a customer to obtain support from that vendor directly.
++
```
Comment (danmacpherson) [inline content/learn/maintained.adoc:61]: Likewise for this one?

File: content/learn/maintained.adoc [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+The {solution-name-upstream} team is will try to address any problems in the {validated-patterns-op}, and in the common Helm charts, but cannot not offer any SLAs at this time.
++
+//TODO: Create an aDoc version of our support statement slide
```
Comment (danmacpherson) [inline content/learn/maintained.adoc:65]: Is this + also necessary since the next line is a comment?

File: content/learn/sandbox.adoc [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+* A {sandbox} pattern must be able to be deployed onto a freshly deployed OpenShift cluster without prior modification or tuning.
+* A {sandbox} pattern must include a top-level README file that highlights the business problem and how the pattern solves it.
+* A {sandbox} pattern must include an architecture drawing. The specific tool or format is flexible as long as the meaning is clear.
+* A {sandbox} pattern must undergo an informal technical review by a community leader to ensure that it meets basic reuse standards.
+* A {sandbox} pattern must undergo an informal architecture review by a community leader to ensure that the solution has the right components, and they are generally being used as intended.
++
+For example, not using a database as a message bus.
+As community leaders, contributions from within {redhat} might be subject to a higher level of scrutiny.
+While we strive to be inclusive, the community will have quality standards and generally using the framework does not automatically imply a solution is suitable for the community to endorse/publish.
```
Comment (danmacpherson) [inline content/learn/sandbox.adoc:51]: Are these two lines separate musts or are they a part of the previous must? If the former, it might split off the last item on the list (line 52). If the latter, you might need +'s here.

File: content/learn/tested.adoc [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+:toc:
+
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="about-tested-tier"]
+= About the {tested-tier-first}
+
+The {tested} tier provides you with additional collateral and reassurance that the pattern was known to be recently working on at least one recent version of {rh-ocp}. Inclusion in this tier requires some additional work for the pattern's owner - which might be a partner or a sufficiently motivated subject matter expert (SME).
```
Comment (danmacpherson) [inline content/learn/tested.adoc:18]:
```suggestion
The {tested} tier provides you with additional collateral and reassurance that the pattern was known to be recently working on at least one recent version of {rh-ocp}. Inclusion in this tier requires some additional work for the pattern's owner, which might be a partner or a sufficiently motivated subject matter expert (SME).
```

File: content/learn/tested.adoc [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
++
+Qualification is a {solution-name-upstream} Technical Oversight Committee (TOC) decision with input from the pattern owner.
+* A {tested} pattern must have their implementation reviewed by the patterns team to ensure that it is sufficiently flexible to function across a variety of platforms, customer environments, and any relevant verticals.
+* A {tested} pattern must include a standardized architecture drawing, created with (or at least conforming to) the standard {solution-name-upstream} tooling
+* A {tested} pattern must include a written guide for others to follow when demonstrating the pattern
+* A {tested} pattern must include a test plan covering all features or attributes being highlighted by the demonstration guide. Negative flow tests (such as resiliency or data retention in the presence of network outages) are also limited to scenarios covered by the demonstration guide.
++
+The test plan must define how to validate if the pattern has been successfully deployed and is functionally operational.
+Example: https://docs.google.com/document/d/12KQhdzjVIsxRURTnWAckiEMB3_96oWBjtlTXi1q73cg/view[Validating an Industrial Edge Deployment]
```
Comment (danmacpherson) [inline content/learn/tested.adoc:55]: Do we need a plus sign before this line?

File: content/learn/tested.adoc [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
++
+//TODO: Convert above link to adoc
```
Comment (danmacpherson) [inline content/learn/tested.adoc:57]: Is the plus sign here necessary due to the next line being a comment?

File: content/learn/tested.adoc [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
```
Comment (danmacpherson) [inline content/learn/tested.adoc:58]:
```suggestion
```

File: content/learn/tested.adoc [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+* A {tested} pattern must nominate at least one currently supported {rh-ocp} release to test against.
+* A {tested} pattern must ensure the test plan passes at least once per quarter.
+* A {tested} pattern must create a publicly available JSON file (eg. in an AWS bucket) that records the result of the latest test for each combination of pattern, platform, and {rh-ocp} version. See link:/learn/test-artefacts/[testing artefacts]
+
+[NOTE]
+====
+The {tested} pattern *do not* imply an obligation of support for partner or community operators by Red Hat or the pattern owner.
```
Comment (danmacpherson) [inline content/learn/tested.adoc:65]:
```suggestion
A {tested} pattern *does not* imply an obligation of support for partner or community operators by Red Hat or the pattern owner.
```

File: content/learn/tested.adoc [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+
+[id="can-tested-tier"]
+=== Can
+
+Teams creating {tested} pattern can provide their own service level agreement (SLA).
+
+A technical document for Quality Engineering (QE) team that defines how to validate if the pattern has been successfully deployed and is functionally operational.
+For example, see https://docs.google.com/document/d/12KQhdzjVIsxRURTnWAckiEMB3_96oWBjtlTXi1q73cg/view[Validating an Industrial Edge Deployment].
+
+* A {tested} pattern meeting additional criteria can be nominated for promotion to the link:/learn/maintained/[Maintained tier].
```
Comment (danmacpherson) [inline content/learn/tested.adoc:82]: Is this bullet point here required since the rest of this section is unbulleted?

File: modules/edd-deploying-edd-pattern.adoc [hunk 16]
```diff
@@ -215,3 +215,11 @@ The most important ArgoCD instance to examine at this point is `emerging-disease
...
 . Check all applications are synchronised. There are thirteen different ArgoCD "applications" deployed as part of this pattern.
 
+[id="viewing-the-sepsis-application-dashboard-getting-started"]
+== Viewing the Sepsis Detection dashboard
+TO-DO: Describe how to examine the various parts of the Sepsis application
+
+= Next Steps
+
+link:https://groups.google.com/g/validatedpatterns[Help & Feedback]
+link:https://github.com/validatedpatterns/emerging-disease-detection/issues[Report Bugs]
```
Comment (danmacpherson) [inline modules/edd-deploying-edd-pattern.adoc:225]: This seems like it should be part of a different PR.

**Review by danmacpherson — COMMENTED** [review]
Hi Avani,

Just found some minor issues. Overall, the content is great. Nice job!

**Comment by abhatt-rh** [comment]
Ignoring the html test errors. These seem to for the renamed files and are sort of false positive. Raised an query with the htmltest repo. Not holding merge for this

**Review by abhatt-rh — COMMENTED** [review]
addresses all feedack
